### PR TITLE
[final] add example

### PR DESCRIPTION
**Pull request by aboedo**
added an example app with instructions

### CODE REVIEW

**Comment by aboedo** [comment]
package-lock.json takes up 12k lines. holy npm. probably good idea to hide it when reviewing

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
 .vscode
-/.idea/
-*.DS_Store
\ No newline at end of file
+*.idea/
+*.DS_Store
+*.log
+examples/cordova-sample/MyApp/node_modules
+examples/cordova-sample/MyApp/platforms
+examples/cordova-sample/MyApp/plugins
```
Comment (vegaro) [inline .gitignore]: it looks like the plugins folder is actually getting added?

Comment (aboedo) [reply .gitignore]: good catch, I added before deciding to ignore, I'll kill it from the repo

**Comment by aboedo** [comment]
ready for another pass

**Comment by aboedo** [comment]
@vegaro bump

**Comment by aboedo** [comment]
@vegaro bump 2

File: examples/cordova-sample/MyApp/config.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <allow-intent href="sms:*" />
+    <allow-intent href="mailto:*" />
+    <allow-intent href="geo:*" />
+    <platform name="android">
+        <allow-intent href="market:*" />
+    </platform>
+    <platform name="ios">
+        <allow-intent href="itms:*" />
+        <allow-intent href="itms-apps:*" />
+    </platform>
```
Comment (vegaro) [inline examples/cordova-sample/MyApp/config.xml]: not sure why but there are some assets for windows phone

Comment (aboedo) [reply examples/cordova-sample/MyApp/config.xml:25]: I've no idea why those got added. there's also like blackberry and others. I'll kill them, hopefully it won't break anything.

**Pull request merged**